e2e/test/scenarios/admin/datamodel/field.cy.spec.js

```diff
@@ -134,39 +133,42 @@ describe("scenarios > admin > datamodel > field", () => {
       cy.contains("Title");
     });
 
-    // [quarantined]: flake, blocking 3rd party PR
-    it.skip("allows 'Custom mapping' null values", () => {
+    it("allows 'Custom mapping' null values", () => {
+      const dbId = 2;
+      const remappedNullValue = "nothin";
+
       restore("withSqlite");
       cy.signInAsAdmin();
-      const dbId = 2;
+
       withDatabase(
         dbId,
-        ({ number_with_nulls: { num }, number_with_nulls_ID }) =>
-          cy.visit(
-            `/admin/datamodel/database/${dbId}/schema/${SAMPLE_DB_SCHEMA_ID}/table/${number_with_nulls_ID}/field/${num}/general`,
-          ),
+        ({ NUMBER_WITH_NULLS: { NUM }, NUMBER_WITH_NULLS_ID }) => {
+          cy.request("GET", `/api/database/${dbId}/schemas`).then(
+            ({ body }) => {
+              const [schema] = body;
+
+              cy.visit(
+                `/admin/datamodel/database/${dbId}/schema/${dbId}:${schema}/table/${NUMBER_WITH_NULLS_ID}/field/${NUM}/general`,
+              );
+            },
+          );
+
+          cy.log("Change `null` to custom mapping");
+          cy.findByRole("heading", { name: "Display values" })
+            .closest("section")
+            .findByText("Use original value")
+            .click();
+          popover().findByText("Custom mapping").click();
+
+          cy.findByDisplayValue("null").clear().type(remappedNullValue).blur();
```

Comment:
I think `.blur()` is unnecessary. When you click the save button in Cypress the `blur` event should be called anyway. And mostly I found we use `blur` when we're autosaving the field, which is not what we do in this case.